docs/topics/whatsnew18.md

```diff
@@ -0,0 +1,62 @@
+[//]: # (title: What's new in Kotlin 1.8.0)
+
+<microformat>
+   <p>The IDE support for Kotlin 1.8.0 is available for IntelliJ IDEA TBD.</p>
+</microformat>
+
+_[Release date: %kotlinReleaseDate%](eap.md#build-details)_
+
+The Kotlin 1.8.0 release is out! Here are some highlights from this release:
+
+* The old backend for Kotlin/JVM has been removed
+* We now support Xcode 14
+* We ensured compatibility with Gradle 7.3
+* We introduced new experimental functions for JVM: recursively copy or delete directory content
+
+## Kotlin/JVM
```

Comment:
Let's move info from the change notes here?

## Kotlin/JVM

* Support for Java 19 bytecode
* Removed old backend (the `-Xuse-old-backend` compiler option is no longer supported)

etc..

Reply:
Or should we do it later?

Reply:
@koshachy, change notes content added!